animation_nodes/nodes/object/lamp_color_input.py

```diff
@@ -0,0 +1,35 @@
+import bpy
+from ... data_structures import Color, ColorList
+from ... base_types import AnimationNode, VectorizedSocket
+
+class LampColorInputNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_LampColorInputNode"
+    bl_label = "Lamp Color Input"
+    useObjectList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput(VectorizedSocket("Object", "useObjectList",
+            ("Object", "object", dict(defaultDrawType = "PROPERTY_ONLY")),
+            ("Objects", "objects"),
+            codeProperties = dict(allowListExtension = False)))
```

Comment:
Those code properties seem redundant since you are not using the vectorization code effect.

Reply:
Okay